incubator/binding-pgsql/src/main/antlr4/io/aklivity/zilla/runtime/binding/pgsql/parser/PostgreSqlParser.g4

```diff
@@ -397,8 +397,8 @@ discardstmt
     ;
 
 altertablestmt
-    : ALTER TABLE (IF_P EXISTS)? relation_expr (alter_table_cmds | partition_cmd)
-    | ALTER TABLE ALL IN_P TABLESPACE name (OWNED BY role_list)? SET TABLESPACE name opt_nowait
+    : ALTER opttable_alter_type (IF_P EXISTS)? relation_expr (alter_table_cmds | partition_cmd)
+    | ALTER opttable_alter_type ALL IN_P TABLESPACE name (OWNED BY role_list)? SET TABLESPACE name opt_nowait
```

Comment:
Seems they are listing out the possibles explicitly, why not just put `ALTER TABLE` and `ALTER TOPIC` here too?

Reply:
I thought since they are logically exactly the same the use like alias

Reply:
Understood, prefer we list them out explicitly if that's okay with you.

Reply:
Sure thing!